relay/nakama/auth/siwe.go

```diff
@@ -80,12 +86,13 @@ func authWithSIWE(
 	logger runtime.Logger,
 	nk runtime.NakamaModule,
 	in *api.AuthenticateCustomRequest,
+	span trace.Span,
```

Comment:
you don't need to be dependency injecting the span around, you can use SpanFromContext (see https://opentelemetry.io/docs/languages/go/instrumentation/#get-the-current-span)